netman/adapters/switches/brocade.py

```diff
@@ -252,7 +252,7 @@ def set_vlan_access_group(self, vlan_number, direction, name):
             if vlan.access_groups[direction] is not None:
                 self.ssh.do("no ip access-group %s %s" % (vlan.access_groups[direction], {IN: 'in', OUT: 'out'}[direction]))
             result = self.ssh.do("ip access-group %s %s" % (name, {IN: 'in', OUT: 'out'}[direction]))
-            if len(result) > 0:
+            if len(result) > 0 and "Warning" not in "".join(result):
```

Comment:
Could it be more like line 87? I'm concerned that since we do not cover all return cases that a case could return something like 'Error: blablabla Warning blablabla..' and so I would verify that it starts with 'Error'... since we want to raise an error when there is an error and not 'when there is something other than a Warning' ... also, this would prevent if there are unsupported case where a 'information message' is returned...